shared/login/signup/request-invite-success/index.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import * as React from 'react'
+import {Text, Icon} from '../../../common-adapters'
+import Wrapper from '../wrapper'
+
+type Props = {|
+  onBack: () => void,
+|}
+
+const RequestInviteSuccess = (props: Props) => (
+  <Wrapper onBack={props.onBack}>
+    <Icon type="icon-invite-code-48" />
+    <Text type="Header">Invite request sent</Text>
+    <Text type="Body">
+      Thanks for requesting an invite to Keybase. When one becomes available, we will send it to you via
+      email.
+    </Text>
```

Comment:
We should probably have a button taking you to the landing screen, and `onBack` should be disabled. Right now going back takes you to the form with email / name input

Reply:
its supposed to reset entirely

Reply:
cool, works now :+1: